src/algorithms/libs/spoofing_detector.h

```diff
@@ -0,0 +1,93 @@
+/*!
+ * \file spoofing_detector.h
+ * \brief Library of anti-spoofing functions
+ * \author Harshad Sathaye sathaye.h(at)northeastern.edu
+ *
+ * -----------------------------------------------------------------------------
+ *
+ * GNSS-SDR is a Global Navigation Satellite System software-defined receiver.
+ * This file is part of GNSS-SDR.
+ *
+ * Copyright (C) 2010-2021  (see AUTHORS file for a list of contributors)
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ * -----------------------------------------------------------------------------
+ */
+
+#ifndef GNSS_SDR_SPOOFING_DETECTOR_H_
+#define	GNSS_SDR_SPOOFING_DETECTOR_H_
+
+#include <iostream>
+#include <fstream>
+#include <list>
+#include <map>
+#include <vector>
+#include <set>
+#include "gnss_synchro.h"
+#include <boost/circular_buffer.hpp>
+#include "configuration_interface.h"
+#include <glog/logging.h>
+#include <gflags/gflags.h>
```

Comment:
don't include what you don't need. Try commenting out all of these includes and see what you actually need. For instance, you don't need `<iostream>` or `<fstream>`. You do need `<map>` but none of the other standard headers. You don't need ``<boost/circular_buffer.hpp>`, nor do you need the glog or gflags includes.